src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,54 @@
+.calendar {
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  justify-content: flex-start;
+  gap: $cell-distance;
+
+  &__day {
+    text-align: center;
+    line-height: $cell-size;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    background-color: #eee;
+    width: $cell-size;
+    height: $cell-size;
+
+    &:hover {
+      transform: translateY(-20px);
+      transition-duration: 500ms;
```

Comment:
move this property from `:hover` to the class for it to work in both cases(when element is hovered and when cursor leaves it)